package.json

```diff
@@ -42,6 +42,11 @@
 		"url": "https://github.com/naver/egjs-component/issues"
 	},
 	"homepage": "https://naver.github.io/egjs-component",
+	"files": [
+		"./*",
+		"declaration/*",
+		"dist/*"
```

Comment:
What about `"src/*"`? Or should it be `"src/**/*"`?

Reply:
To sum up with the comments above, this fixes two major things.

First, it prevents the omission of essential folders for packages, such as `dist` folder, in certain deployment environments. The issued one was Mac OS 12 Monterey with Node 14​.
I also excluded the `src` folder that is deployed with this package because I thought it was included unnecessarily. Should the `src` folder be deployed together?

Reply:
To think of it, I think it's fine to omit it either.
Although we've included `src` before in our previous build. as our other component usually packs src in `pacakges/vanilla_comp` it would be too verbose for all components to include like that.
It's not the case for `@egjs/component` though.